polaris.shopify.com/pages/examples/index-table-small-screen-with-all-of-its-elements.tsx

```diff
@@ -119,8 +119,8 @@ function SmallScreenIndexTableWithAllElementsExample() {
             {name}
           </Text>
           <p>{location}</p>
-          <p>{orders}</p>
-          <p>{amountSpent}</p>
+          <Text variant="bodyMd" as="p" alignment="end">{orders}</Text>
+          <Text variant="bodyMd" as="p" alignment="end"></Text>{amountSpent}</Text>
```

Comment:
```suggestion
          <Text variant="bodyMd" as="p" alignment="end">{amountSpent}</Text>
```
There was an extra closing </Text> that was causing issues. This should resolve failing CI checks